2-mandatory/6-journey-planner.js

```diff
@@ -121,10 +129,8 @@ function getLocationName() {}
 
   Advanced challange: try to use arrow function when invoking an array method.
 */
-function journeyPlanner(locations, transportMode) {
-  // Implement the function body
-}
-
+ const journeyPlanner = (locations, transportMode) =>locations.filter((element) => element.includes(transportMode))
+      .map((element) => getLocationName(element));
```

Comment:
this is a really concise solution well done, good use chaining methods & use of previous function too